tests/testAutopas/tests/cells/SortedCellViewTest.h

```diff
@@ -0,0 +1,15 @@
+/**
+ * @file SortedCellViewTest.h
+ * @author C. Menges
+ * @date 26.05.2019
+ */
+
+#pragma once
+
+#include <gtest/gtest.h>
+#include <testingHelpers/commonTypedefs.h>
```

Comment:
```suggestion
#include "testingHelpers/commonTypedefs.h"
```
and then resort the includes